server/converters/cxgtool.py

```diff
@@ -114,6 +114,13 @@ def main():
         help="URL providing more information about the dataset (hint: must be a fully specified absolute URL).",
     )
     parser.add_argument("--out", "--output", "-o", help="output CXG file name")
+    parser.add_argument(
```

Comment:
one idea triggered by this work -- it may be useful to add a stand-alone validation tool that compares a CXG to an H5AD and complains if they don't match.   It could be used as part of creating a new CXG to confirm correctness, and as a stand-alone option used to validate the integrity of pre-existing CXGs (eg, does the version on S3 match the H5AD) and would make our unit tests more robust.

Reply:
Yeah, that's a good idea.   Mind if I add that as a separate PR?

Reply:
Definitely a separate PR.